### PR TITLE
Make a doc-only change

### PR DESCRIPTION
**Pull request by RikkiGibson**

### CODE REVIEW

File: docs/contributing/Building, Debugging, and Testing on Unix.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Particularly for developers who aren't experienced with .NET Core development on
 1. Clone git@github.com:dotnet/roslyn.git
 1. Run `./build.sh --restore`
 1. Run `./build.sh --build`
+2. Don't forget to update libssh!
```
Comment (Youssef1313) [inline docs/contributing/Building, Debugging, and Testing on Unix.md:10]: Markdown already auto numbers these.

```suggestion
1. Don't forget to update libssh!
```

Comment (RikkiGibson) [reply docs/contributing/Building, Debugging, and Testing on Unix.md:10]: The actual content of the doc change is not important here, just testing that we are able to go through a fast path in CI :smile:

Comment (Youssef1313) [reply docs/contributing/Building, Debugging, and Testing on Unix.md:10]: Ah I didn't see the branch name "dont-build-docs-changes" 😄 
Hope things work well 🎉

**Comment by RikkiGibson** [comment]
I think the changes related to this PR are too far out of date to be usable. Maybe we can revisit this question after we are done working on getting CI to just generally run faster.

**Pull request closed without merging**